docs/ASM.md

```diff
@@ -209,16 +209,16 @@ The quick create option requires the following options for a windows instance:
                 --azure-dns-name 'myserverdnsname'
                 --azure-service-location 'West US'
                 --azure-source-image 'windows-2012-image-id'
-                --winrm-user 'jetstream'
-                --winrm-password 'jetstream@123'
+                --connection-user 'jetstream'
+                --connection-password 'jetstream@123'
                 --distro 'windows-chef-client-msi'
```

Comment:
Could you please check are we still supporting --distro option ? Because I guess this has been deprecated a long ago and now chef-15 do no have this option.